src/dawn/CodeGen/GridTools/GTCodeGen.cpp

```diff
@@ -783,6 +785,7 @@ void GTCodeGen::generateStencilClasses(
     // Add static asserts to check halos against extents
     StencilConstructor.addComment("Check if extents do not exceed the halos");
     int nonTempFieldId = 0;
+    std::map<std::string, iir::Extents> fieldsExtsMap;
```

Comment:
I would suggest not to use arbitrary truncation. If you look around there are many function calls with names as long as `insertStatementAccessesPair`. The main goal here should be readability and since there is no downside to naming things properly I'd sugest the map to be called `parameterTypteToFullExtentMap`

Reply:
done